src/bundler.ts

```diff
@@ -24,6 +25,60 @@ interface BundleOptions {
   onBeforeDownload?: LifecycleHook
 }
 
+interface CommonBundleOptions {
```

Comment:
Not a huge deal, but do you think we could use the same interface for both formats, even if some properties end up only being used by one of the formats?

Reply:
I did wonder if that might be better actually, I'll go with that, for sure